references/recognition/README.md

```diff
@@ -64,3 +64,15 @@ Feel free to inspect the multiple script option to customize your training to yo
 ```python
 python references/recognition/train_pytorch.py --help
 ```
+## Using custom fonts
+If you want to use your own custom fonts for training, make sure the font is installed on your OS. 
+You can do so in linux by either:
+* clicking on the .ttf file and pressing the install button.  
```

Comment:
I think we can remove this point Pillow seems not to look inside ~/.fonts :) so this will not work